interactive/views.py

```diff
@@ -70,6 +70,16 @@ def new_analysis_request_done(request):
     return render(request, "interactive/new_analysis_request_done.html")
 
 
+@login_required
```

Comment:
Are we happy to allow any logged in user to see this?

Reply:
Nope, good spot, completely forgot about that. Might raise a separate ticket, since it's probably not one line fix. Needs to be the requester or an admin only.

Reply:
https://github.com/opensafely-core/interactive.opensafely.org/issues/124